packages/combine-adapters/package.json

```diff
@@ -4,9 +4,6 @@
   "description": "An adapter which combines other adapters for flipflop",
   "main": "dist/flopflip-combine-adapters.cjs.js",
   "module": "dist/flopflip-combine-adapters.esm.js",
-  "scripts": {
-    "prepare": "./../../bin/version.js amend"
```

Comment:
The `prepare` script is not executed anymore by Yarn v3.

https://yarnpkg.com/advanced/lifecycle-scripts